### DISCUSSION TITLE
Thrusters thrusting.

### DISCUSSION THREAD

**Discussion by RedigTuffing** [tsunamayo/Starship-EVO#5116] [Suggestions]
I feel that the thrusters lack a bit of motion to them. I suggest that thrusters have a stream or a "flame" coming out of it when said thruster is affecting the trajectory. Not constantly but rather only when it is actually thrusting.

Hope I make sense 😊👍🏻

**Comment by Kaiser-Indrasil** [comment]
@RedigTuffing That actually doesn't make sense, since this exact functionality is already in the game.
Granted, you have to switch to the Experimental branch within Steam game's settings.
The [game's official Discord server](https://discord.gg/starshipevo) is the best place to get your questions answered the fastest.

![image](https://github.com/tsunamayo/Starship-EVO/assets/63756057/35d524b1-b34a-422a-83fe-d8d8b618eff3)

**Reply by RedigTuffing** [reply]
Oh, I had no idea!
Thank you for informing me. I appreciate it.
